lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -498,6 +498,12 @@ sub overview {
     }
 
     if (!$search_args{build}) {
+        if (@groups > 1) {
+            return $self->render(text => 'Specify a build when you want to lookup multiple groups', status => 404);
```

Comment:
Ah, no. rendering text on HTML routes is not an option. HTML routes uses flashes for errors